### PR TITLE
Upgraded readme.md

### PR DESCRIPTION
**Pull request by scheyal**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,43 +1,67 @@
...
+
+BF is based on the Node.js platform and the [OClif](https://github.com/oclif/oclif) framework where it inherits its command line parsing style, and plugin architecture platform. 
+
+You must download the following prerequisites:
+
+* [Node.js](https://nodejs.org/) version 10.14.1 or higher
+
+Install the tool using the following command: 
+
+~~~
```
Comment (christopheranderson) [inline README.md]: I think you can bump up the code snippet. Microsoft's Voice guidance is to get to the point, especially with dev content.

File: README.md [hunk 2]
```diff
@@ -1,43 +1,67 @@
...
+
 # BF Command Line Interface
 
 [![Build Status](https://fuselabs.visualstudio.com/SDK_v4/_apis/build/status/CLI/Botframework-CLI-CI-PR?branchName=master)](https://fuselabs.visualstudio.com/SDK_v4/_build/latest?definitionId=537&branchName=master)
 [![Coverage Status](https://img.shields.io/coveralls/github/microsoft/botframework-cli/master)](https://coveralls.io/github/microsoft/botframework-cli?branch=master)
 
 The new BF Command Line Interface (CLI) tool replaces the collection of standalone tools used to manage Bot Framework bots and related services. We have ported most tools and are in process of porting the rest. The new BF CLI aggregates the collection of cross-platform tools into one cohesive and consistent interface.
 
-The old tools will be deprecated in subsequent releases. All new investments, bug fixes, and new features will be implemented in the new consolided BF CLI alone.
+The old tools will be deprecated in subsequent releases. All new investments, bug fixes, and new features will be implemented in the new consolidated BF CLI alone.
```
Comment (christopheranderson) [inline README.md]: Can this be moved to the overview section?

**Comment by scheyal** [comment]
Fixes #240

**Comment by scheyal** [comment]
Fixes #242

**Pull request merged**